### PR TITLE
Upgrade comit-sdk to 0.15.0

### PR DESCRIPTION
**Pull request by D4nte**
Draft until https://github.com/comit-network/comit-js-sdk/pull/169 is merged

### CODE REVIEW

File: api_tests/src/actors/defaults.ts [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+export function defaultHalightLightningBitcoinHanEthereumEther(
+    lndPubkey: string,
+    peer: Peer
+): HalightLightningBitcoinHanEthereumEtherRequestBody {
+    return {
+        alpha_amount: "10000000",
+        beta_amount: "5000000000000000000",
+        alpha_network: "regtest",
+        beta_network: "17",
```
Comment (D4nte) [inline api_tests/src/actors/defaults.ts]: Are we happy with that or do we want to update the current design?

File: api_tests/src/actors/actor.ts [hunk 2]
```diff
@@ -238,11 +242,22 @@ export class Actor {
             throw new Error("Cannot deploy htlc for nonexistent swap");
         }
 
-        const txid = await this.swap.deploy(Actor.defaultActionConfig);
+        const transaction = await this.swap.deploy(Actor.defaultActionConfig);
+        let transactionId;
+        if (transaction instanceof Transaction) {
+            const status = await transaction.status();
+            transactionId = transaction.id;
+            if (!status) {
```
Comment (thomaseizinger) [inline api_tests/src/actors/actor.ts]: Please check against the actual enum an not a number.

File: api_tests/src/actors/defaults.ts (hunk 1 above)
Comment (thomaseizinger) [reply api_tests/src/actors/defaults.ts]: Let's roll with this for now.

**Comment by D4nte** [comment]
Sorry, I should not have review request on this. https://github.com/comit-network/comit-js-sdk/pull/169 should be sorted first.

File: api_tests/src/actors/actor.ts [hunk 3]
```diff
@@ -238,11 +243,28 @@ export class Actor {
             throw new Error("Cannot deploy htlc for nonexistent swap");
         }
 
-        const txid = await this.swap.deploy(Actor.defaultActionConfig);
+        const transaction = await this.swap.deploy(Actor.defaultActionConfig);
+        let transactionId;
+        if (transaction instanceof Transaction) {
+            {
+                // Temporary hack until comit-sdk can return a status for unconfirmed failed transactions
+                await this.wallets.ethereum.getTransactionStatus(
```
Comment (thomaseizinger) [inline api_tests/src/actors/actor.ts]: The return value is not assigned to anything?

**Pull request merged**